config.yaml

```diff
@@ -440,3 +479,109 @@ metrics:
     chains: [celo]
     variants:
       - [Reserve]
+
+  # Track total supply of all stable tokens (once per minute for cost efficiency)
+  - source: cUSD.totalSupply()(uint256)
+    schedule: 0 * * * * *
```

Comment:
If we're just using this for the dashboard and not any alerting, do we need the calls this frequent? Would once an hour not suffice?

(1440 * 15) calls vs (24 * 15) calls